README.md

```diff
@@ -307,6 +328,18 @@ public interface ExampleHandler {
 }
 ```
 
+You also can make your handler extended from some other handler class by
```

Comment:
Class - no, interface - yes but it'll parse it as if it was ManualHandler. I may add support for it in the future but I'm not sure if it's a good usecase.